l10n_jp_address_layout/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
Do we need this file?

Reply:
@TimLai125

I'm using vim and the default indent of vim is tab.
So, I need to take care about indent size when I edit code...

editorconfig is not only for vim editor.
Almost all modern IDE support this file within itself or some plugin.
I think editorconfig helps any developers work.

But If you don't want to include this file within this repo,
I'll rebase to remove this patch from change-sets of this PR.

Anyway, thank u for reviewing my PR.

Reply:
@eternalharvest Could you please remove .editorconfig file since something for IDE/editor is outside the scope of this repository.  No need to rebase this time although it's also fine if you do so.